vtn/src/test/java/com/lftechnology/vtn/api/TransactionApiTest.java

```diff
@@ -0,0 +1,34 @@
+package com.lftechnology.vtn.api;
+
+import com.lftechnology.vtn.dto.request.Credential;
+import com.lftechnology.vtn.dto.request.TransactionRequest;
+import com.lftechnology.vtn.dto.response.TransactionResponse;
+import org.junit.Test;
+
+import java.math.BigDecimal;
+import java.time.LocalDate;
+import java.util.Date;
+
+import static org.junit.Assert.*;
+
+public class TransactionApiTest {
+
+    @Test
```

Comment:
we are using live credentials...and if the txn is successful it will deduct amount from the account balance...